components/avatar-component/AvatarComponent.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div>
+    <img v-bind:src="avatarSrc">
+  </div>
+  </template>
+
+<script>
+import '~/components/colors/colors.css'
+import _defaultAvatarSrc from './assets/Portrait_Placeholder.png'
+
+export default {
+  props: {
+    avatar: {
+      type: Object
+    },
+    name: {
+      type: String
+    }
+    // 'storytellerAvatar',
+    // 'storyteller',
+  },
+  computed: {
+    avatarSrc: function () {
```

Comment:
you can make this shorthand `avatarSrc () {` (or `imgSrc () {` if you had changed the aforementioned name

Reply:
naturally after changing the above